src/content/content_recommender.py

```diff
@@ -1,18 +1,34 @@
+import numpy as np
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.metrics.pairwise import cosine_similarity
 class ContentRecommender:
 
     def __init__(self):
-        pass
+        self.word_tfidf = TfidfVectorizer(ngram_range=(1, 1), analyzer='word', sublinear_tf=True, max_df=0.5, stop_words='english')
 
 #TODO - Fit - form a vocab using training events(all valid events).
 #TODO - Ideas for vocab (1 gram, 2 gram or ngram).
     def train(self, training_events):
         ## input : dict([user, list of events])
         ## output : dict([user, tfidf - vector])
+        # Assumed training events to be a list of event_info objects
+        training_events = np.array([event.description for event in training_events])
```

Comment:
training events as we discussed yesterday was dict([user, list of events])? It was for all users, right? Are we now assuming, it is for single user? Why? Shouldn't we do for all users? 

Also, as far as I remember from yesterday discussing, we planned not to use data model classes for now. We are directly going to use the dictionaries we made in pre - processing step.